.github/workflows/build-complete-samples.yml

```diff
@@ -436,6 +436,10 @@ jobs:
               name: 'bot-sso-adaptivecard'
               version: '6.0.x'
 
+            - project_path: 'samples/meetings-app-icon-badging/csharp/InMeetingNotifications/InMeetingNotificationsBot.csproj'
```

Comment:
I can't see deployment pipeline here.

Also, run build pipeline and add UI screenshot.